docs/1.8.9/chattriggers.md

```diff
@@ -0,0 +1,188 @@
+
+### AzuredClient
+
+#### General
+
+* Chat Emotes (Hypixel's) - ?
+* Mastery Helper - ?
+* Party Commands - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Background Blur - [tterrag's Blur](https://www.curseforge.com/minecraft/mc-mods/blur/files?version=1.8.9)
+
+#### Dungeons
+
+* Announce Melody - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Announce Melody Progress - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Terminal Timer (Saves your PBs for each terminal) - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Relic Timer - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Send message on Dungeon Death (customizable) - ?
+
+* Invincibility Timers - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Bonzo and Phoenix Announce - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Necron Invincibility Timer - ?
+* Goldor Tick Timer - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Leap Announce - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Drag Split Notifications - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+
+* Highlight Keys - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Show Blessings (Display for dungeon blessings on screen) - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Highlight on Leap Menu (based on class) - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+
+#### HUD
+* Ping Display - [Polyfrost's EvergreenHUD](https://modrinth.com/mod/evergreenhud) / [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* TPS Display - [Polyfrost's EvergreenHUD](https://modrinth.com/mod/evergreenhud) / [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* FPS Display - [Polyfrost's EvergreenHUD](https://modrinth.com/mod/evergreenhud) / [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* BPS Display - [Polyfrost's EvergreenHUD](https://modrinth.com/mod/evergreenhud) / [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Customizable colors - ?
+
+#### Garden
+* Visitor List Display - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Milestone Display - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Show crop milestones as stack size - ?
+* Show composter upgrades as stack size - ?
+
+#### Render
+* Hide Falling Blocks - [Polyfrost's OverflowParticles](https://modrinth.com/mod/overflowparticles)
```

Comment:
Are you sure this is OverflowParticles and not PolyPatcher?

Reply:
I can confirm it is in overflowparticles

Reply:
![image](https://github.com/user-attachments/assets/1b1ae475-8a50-4fc8-8300-4e63dd280db7)
is this not it?

Reply:
Micro is right on this. It's in PolyPatcher and not OverflowParticles

Reply:
good cuz otherwise ive gone schizo. im the one who PRd the feature

Reply:
Oh i misread lmfao